test/infra/vsphere/export.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+# shellcheck disable=SC2001
+echo "${SSH_BASTION_KEY_CONTENTS}" | sed 's/\\n/\n/g' >> vsphere-tests.pem
```

Comment:
should we generate a temp keypair if CI does not provide `SSH_BASTION_KEY_CONTENTS` ?